### ISSUE TITLE
Unable to locate element: {"method":"css selector","selector":"div[aria-haspopup="menu"]"}

### ISSUE THREAD

**Issue by Jaheay** [iamseyedalipro/ChatGPTAutomation#6]
Steps to replicate: 

1) Download chrome for testing to folder
2) Download chromedriver to folder. 

Setup relative paths: 

```
chat_bot = ChatGPTAutomation(
    chrome_path=".\\chrome\\chrome.exe", 
    chrome_driver_path=".\\chromedriver\\chromedriver.exe",
    )
   
   
chat_bot.switch_model("4")
chat_bot.send_prompt_to_chatgpt("Hello, ChatGPT!")
chat_bot.save_conversation(CONVERSATION_DIR + "conversation.txt")
```
    
```
Traceback (most recent call last):
  File "D:\Code\code-chatgpt\main.py", line 14, in <module>
    chat_bot.switch_model("4")
  File "D:\Code\code-chatgpt\.venv\Lib\site-packages\chatgpt_automation\chatgpt_automation.py", line 702, in switch_model
    menu_element = self.driver.find_element(*ChatGPTLocators.CHATGPT_SWITCH_HOVER_BTN)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Code\code-chatgpt\.venv\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 831, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Code\code-chatgpt\.venv\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 440, in execute
    self.error_handler.check_response(response)
  File "D:\Code\code-chatgpt\.venv\Lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 245, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"div[aria-haspopup="menu"]"}
  (Session info: chrome=131.0.6778.108)
Stacktrace:
        GetHandleVerifier [0x00007FF7ABDD6CC5+28821]
        (No symbol) [0x00007FF7ABD43850]
        (No symbol) [0x00007FF7ABBE578A]
        (No symbol) [0x00007FF7ABC391BE]
        (No symbol) [0x00007FF7ABC394AC]
        (No symbol) [0x00007FF7ABC82647]
        (No symbol) [0x00007FF7ABC5F33F]
        (No symbol) [0x00007FF7ABC7F412]
        (No symbol) [0x00007FF7ABC5F0A3]
        (No symbol) [0x00007FF7ABC2A778]
        (No symbol) [0x00007FF7ABC2B8E1]
        GetHandleVerifier [0x00007FF7AC10FCCD+3408029]
        GetHandleVerifier [0x00007FF7AC12743F+3504143]
        GetHandleVerifier [0x00007FF7AC11B61D+3455469]
        GetHandleVerifier [0x00007FF7ABE9BDCB+835995]
        (No symbol) [0x00007FF7ABD4EB6F]
        (No symbol) [0x00007FF7ABD4A824]
        (No symbol) [0x00007FF7ABD4A9BD]
        (No symbol) [0x00007FF7ABD3A1A9]
        BaseThreadInitThunk [0x00007FFCF0ADE8D7+23]
        RtlUserThreadStart [0x00007FFCF127FBCC+44]

```
